src/Microsoft.EntityFrameworkCore.Tools.DotNet/Properties/Resources.resx

```diff
@@ -120,33 +120,168 @@
   <data name="BuildFailed" xml:space="preserve">
     <value>Build failed.</value>
   </data>
+  <data name="ConfigurationDescription" xml:space="preserve">
+    <value>The configuration to use.</value>
+  </data>
+  <data name="ConnectionDescription" xml:space="preserve">
+    <value>The connection string to the database.</value>
+  </data>
+  <data name="ContextDescription" xml:space="preserve">
+    <value>The DbContext to use.</value>
+  </data>
+  <data name="ContextNameDescription" xml:space="preserve">
+    <value>The name of the DbContext.</value>
+  </data>
+  <data name="DataAnnotationsDescription" xml:space="preserve">
+    <value>Use attributes to configure the model (where possible). If omitted, only the fluent API is used.</value>
+  </data>
+  <data name="DatabaseDescription" xml:space="preserve">
+    <value>Commands to manage the database.</value>
+  </data>
+  <data name="DatabaseDropDescription" xml:space="preserve">
+    <value>Drops the database.</value>
+  </data>
+  <data name="DatabaseDropDryRunDescription" xml:space="preserve">
+    <value>Show which database would be dropped, but don't dorp it.</value>
+  </data>
+  <data name="DatabaseDropForceDescription" xml:space="preserve">
+    <value>Don't confirm.</value>
+  </data>
+  <data name="DatabaseUpdateDescription" xml:space="preserve">
+    <value>Updates the database to a specified migration.</value>
+  </data>
+  <data name="DbContextDescription" xml:space="preserve">
+    <value>Commands to manage DbContext types.</value>
+  </data>
+  <data name="DbContextInfoDescription" xml:space="preserve">
+    <value>Gets information about a DbContext type.</value>
+  </data>
+  <data name="DbContextListDescription" xml:space="preserve">
+    <value>Lists available DbContext types.</value>
+  </data>
+  <data name="DbContextScaffoldDescription" xml:space="preserve">
+    <value>Scaffolds a DbContext and entity types for a database.</value>
+  </data>
+  <data name="DbContextScaffoldForceDescription" xml:space="preserve">
+    <value>Overwrite existing files.</value>
+  </data>
+  <data name="DotnetEfFullName" xml:space="preserve">
+    <value>Entity Framework Core .NET Command Line Tools</value>
+  </data>
+  <data name="EFFullName" xml:space="preserve">
+    <value>Entity Framework Core Command Line Tools</value>
+  </data>
+  <data name="EnvironmentDescription" xml:space="preserve">
+    <value>The environment to use. Defaults to "Development".</value>
+  </data>
+  <data name="FrameworkDescription" xml:space="preserve">
+    <value>The target framework.</value>
```

Comment:
Hmm... probably not since it's an advanced scenario and those who have to use it will know what to put here (the same thing they passed to `dotnet build`)